index.html

```diff
@@ -2072,20 +2072,18 @@ <h3>Extensibility</h3>
           <h4>Semantic Interoperability</h4>
 
           <p>
-This specification endeavors to enable both the JSON and JSON-LD syntaxes to
-be semantically compatible with one another without the JSON implementations
-needing to process the documents as JSON-LD. To achieve this, the
-specification imposes the following additional restrictions on both syntaxes:
+This specification ensures that "plain" JSON and JSON-LD syntaxes are semantically
```

Comment:
I like this language better than the language in my PR, I'll move it over to my PR in an attempt to align the language.

Reply:
Applied in https://github.com/w3c/vc-data-model/pull/546/commits/0221fbb2eb2833505848649e65980e38110d0f55.